src/Components/Search/Search.jsx

```diff
@@ -0,0 +1,87 @@
+// Search.js
+import React, { useState } from "react";
+import { FaSearch } from "react-icons/fa";
+import axios from "axios";
+import Ruselt from "../Ruselt/Ruselt";
+import "./search.css";
+
+
+export default function Search() {
+  const [query, setQuery] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+  const [noResults, setNoResults] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const handleSearch = async (e) => {
+    e.preventDefault();
+
+    try {
+      setLoading(true);
+
+      const response = await axios.get(
+        `https://api.spoonacular.com/recipes/complexSearch?query=${query}&apiKey=684e5784b0184b8ca8ca75da8ef075e1`
```

Comment:
store the api key in a variable in api folder and use the variable in each api request